### ISSUE TITLE
Highlighted entry in Plugins Admin with dark mode not distinguishable

### ISSUE THREAD

**Issue by CennoxX** [notepad-plus-plus/notepad-plus-plus#11331]
### Description of the Issue
When using dark mode a searched highlighted entry in plugins admin is very difficult to recognize, in light mode it is easy viewable.

### Steps to Reproduce the Issue
1. use dark mode
2. open plugins admin
3. search a plugin
4. compare with light mode

### Expected Behavior
The highlighted entry should be distinguishable from the not highlighted entries

### Actual Behavior
The highlight is very difficult to distinguish from the rest

![grafik](https://user-images.githubusercontent.com/6975881/156540055-54e87a8d-e597-4b65-bf15-a0148688a464.png)

compare that to light mode:
![grafik](https://user-images.githubusercontent.com/6975881/156539655-471185a8-3ca7-4e65-8f80-f06a76112ebb.png)

Notepad++ v8.3.2   (32-bit)
Build time : Feb 26 2022 - 23:11:29
Path : C:\Users\Chromwell\Documents\Tools\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 2009
OS Build : 19043.1526
Current ANSI codepage : 65001
Plugins : BetterMultiSelection.dll ComparePlugin.dll CustomizeToolbar.dll LuaScript.dll mimeTools.dll NppCalc.dll NppConverter.dll NppExport.dll NppMenuSearch.dll

**Comment by thomassmith311** [comment]
I'm getting something different:

![image](https://user-images.githubusercontent.com/19561856/156625122-9d804188-b50d-4b87-8cd6-839644f77ea3.png)

Notepad++ v8.3.2   (64-bit)
Build time : Feb 27 2022 - 03:35:46
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 11 (64-bit) 
OS Version : 2009
OS Build : 22000.493
Current ANSI codepage : 1252
Plugins : mimeTools.dll NppConverter.dll NppExport.dll

**Comment by rdipardo** [comment]
@thomassmith311,
> if the plugin admin window is selected, then the selection is in fact blue

The Screen Snip Windows utility doesn't seem to capture the pointer, but I can only reproduce that style by hovering the mouse:

![hover](https://user-images.githubusercontent.com/59004801/156686545-628bc1ee-8bb5-4933-95de-b4c68b58f048.png)

Without hovering, even a solitary active window (note the bright blue title bar) looks just like @CennoxX's original posting:

![no_hover](https://user-images.githubusercontent.com/59004801/156687884-62a834d8-1547-44e8-b09c-20ccac979093.png)

**Comment by CennoxX** [comment]
> The Screen Snip Windows utility doesn't seem to capture the pointer, but I can only reproduce that style by hovering the mouse:

Same for me. I can also see the selection, if I resize the columns.

Okay, I didn't know this: the background color of the plugin list is set by the currently used style. I tested it with DarkModeDefault before, but I didn't restartet Notepad++ and Plugins Admin seems to not update its style without restart. But still, even with DarkModeDefault the highlight is harder to see than with Light Mode.

**Comment by thomassmith311** [comment]
@rdipardo It does it for me even without hovering, it just has to be the primary window. Do you have Windows 11? However, the "looks just like" screenshot you showed does *not* look just like Cennox's screenshot. In Cennox's, it's very hard to distinguish. In yours, it's noticeably easier. And the one you showed looks just like what I showed.

@CennoxX Agreed, it is still harder to see, though definitely easier than the screenshot you originally posted. But it should be possible to change by editing DarkModeDefault.xml, right? Specifically, at the bottom under Global Styles?

**Comment by rdipardo** [comment]
@thomassmith311,
> It does it for me even without hovering, it just has to be the primary window.

I stand corrected; it does highlight the active selection when the window is active, not just on mouse hover:

![selected](https://user-images.githubusercontent.com/59004801/156862515-7762e169-90b5-48d5-8c62-aacb0caaadc7.png)

> Do you have Windows 11? 

Yes; theme is "DarkModeDefault";

~~~
Notepad++ v8.3.2   (64-bit)
Build time : Feb 24 2022 - 16:15:43
Path : C:\Users\Rob\bin\npp_8.3.2\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : ON
Cloud Config : OFF
OS Name : Windows 11 (64-bit) 
OS Version : 2009
OS Build : 22000.527
Current ANSI codepage : 65001
Plugins : HTMLTag.dll mimeTools.dll NppConverter.dll NppExport.dll 
~~~

> the "looks just like" screenshot you showed does not look just like Cennox's screenshot.

A difference in DPI could account for the lower contrast of the latter (I used [LICECap](https://www.cockos.com/licecap/) because it captures the mouse pointer unlike Screen Snip). @Cennox's title bar is also not lit up in the Windows UI's accent colour; manually selecting the window might emphasize the dialog controls a bit more.
 
> it should be possible to change by editing DarkModeDefault.xml, right? Specifically, at the bottom under Global Styles?

Dialog control styles have a dedicated menu at `Settings > Preferences > Dark Mode`:

![dm_styler_menu](https://user-images.githubusercontent.com/59004801/156862626-a6e36a1c-495e-4338-9247-34271b7d5441.png)

I did not find it to be very intuitive or useful (more indication of dark mode's ongoing "beta" status). Here's the result of choosing bright fuschia for the "Active" tone:  

![custom_active](https://user-images.githubusercontent.com/59004801/156862642-8e47a183-9ccb-4a86-81bd-c1bf5a34abfa.png)

Haven't looked at `DarkModeDefault.xml` yet.

**Comment by thomassmith311** [comment]
Only @CennoxX can confirm this but it looks like he has Windows 10, which could account for the color differences we have.

**Comment by rdipardo** [comment]
#11342 takes aim at DM font resolution, but it might be useful to revisit this issue after/if it gets merged.

**Comment by CennoxX** [comment]
@thomassmith311 As stated above, the color difference is due to another style I am using. If I use DarkModeDefault it looks exactly like your screenshots. But yes, I use Windows 10.

**Comment by thomassmith311** [comment]
I was thinking that the blue highlight color was something with windows 11.
________________________________

**Comment by CennoxX** [comment]
This was not fixed with  #11342, please reopen this issue.
